package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "@flowsquad/sarif-report-parse",
-  "version": "1.0.2",
+  "name": "@miragon/sarif-report-parse",
+  "version": "2.0.0",
```

Comment:
1.0.3?

Reply:
Habe das in Teams erklärt... (PR würde geschlossen ohne merge)